xwiki-platform-core/xwiki-platform-security/pom.xml

```diff
@@ -26,8 +26,9 @@
     <version>13.7-SNAPSHOT</version>
   </parent>
   <artifactId>xwiki-platform-security</artifactId>
-  <name>XWiki Platform - Security</name>
-  <description>XWiki Platform - Security is a replacement for the old xwiki rights service (experimental)</description>
+  <name>XWiki Platform - Security - Parent POM</name>
+  <description>XWiki Platform - Security is a replacement for the old xwiki rights service (experimental)
```

Comment:
Side note: we need to remove "experimental". Can be done later on but we need to remember it.

Reply:
Can't indeed hardly be called "experimental" anymore :)